dasharo-security/secure-boot.robot

```diff
@@ -233,6 +234,57 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
     Select File In File Explorer    DB.txt
     Read From Terminal Until    ERROR: Unsupported file type!
 
+SBO009.001 Check automatic certificate provisioning
+    [Documentation]    This test verifies that the automatic certificate
+    ...    provisioning will install custom keys which will make Ubuntu
+    ...    unbootable.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO009.001 not supported
+    Power On
+    # Erase Secure Boot Keys
+    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
+    Enable Secure Boot    ${sb_menu}
+    Save Changes
+    Reenter Menu
+    ${sb_menu}=    Get Secure Boot Menu Construction
+    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
+    Erase All Secure Boot Keys    ${advanced_menu}
+    Save Changes And Reset    3    5
+    # Reboot to Automatic provisioning
+    Boot Dasharo Tools Suite    USB    True
+    Boot System Or From Connected Disk    ubuntu
+
+    Read From Terminal Until    Press any key to continue...
+
+SBO009.002 Check automatic certificate provisioning KEK certificate
+    [Documentation]    This test verifies that the automatic certificate
+    ...    provisioning installs the expected KEK certificate.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO009.002 not supported
+    Power On
+    Boot Dasharo Tools Suite    USB
+    Enter Shell In DTS
+    # Get the KEK certificates
+    Execute Command In Terminal    mokutil --kek > mokutil-kek
+    Execute Command In Terminal
+    ...    wget https://raw.githubusercontent.com/Wind-River/meta-secure-core/master/meta-signing-key/files/uefi_sb_keys/KEK.crt
+    Execute Command In Terminal    openssl x509 -in KEK.crt -noout -text -fingerprint > kek
+
+    # mokutil and openssl-x509 have slightly different output formats. They are
+    # therefore compared using a python keyword.
+    ${mokutil_file}=    Execute Command In Terminal    cat mokutil-kek
```

Comment:
@PLangowski can't we here just run `mokutil --kek` ? This way we could remove the `Execute Command In Terminal mokutil --kek > mokutil-kek` from couple lines above

Reply:
Amended here a52b5a7077b3a2c84403ec7265a3f8eabff285b7.